src/tools/miri/src/diagnostics.rs

```diff
@@ -401,30 +401,32 @@ pub fn report_msg<'tcx>(
     } else {
         // Make sure we show the message even when it is a dummy span.
         for line in span_msg {
-            err.note(&line);
```

Comment:
I think that these changes are best addressed in PRs on their respective repos. Clippy and Miri aren't developed in-tree, they're just pulled regularly.

Reply:
You sure about that? I've done many PRs where I changed clippy and miri and rustfmt because of API changes to compiler code, and I'm following the same process here.

Reply:
Oh right, this isn't just fixing a lint; there's an API change involved. My mistake sorry.

Reply:
Yeah, it's definitely _courteous_ to open the PR to them first if you can be arsed but the reason we switched to subtrees is precisely so that the development flow can be two-way (which it can't be with a submodule).

Reply:
Yeah, this is a case where I think modifying in r-l/rust is fine.